src/main/java/testSystem/App.java

```diff
@@ -0,0 +1,15 @@
+package testSystem;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+
+@SpringBootApplication
+@EnableSwagger2
```

Comment:
You can move this annotation on the SwaggerConfig class. And annotations should be placed like a pyramid:
```java
@EnableSwagger2
@SpringBootApplication
public class App { ...
```